app/keepers.go

```diff
@@ -220,7 +220,6 @@ func (app *OsmosisApp) InitNormalKeepers() {
 	app.IncentivesKeeper = incentiveskeeper.NewKeeper(
 		appCodec, keys[incentivestypes.StoreKey],
 		app.GetSubspace(incentivestypes.ModuleName),
-		*app.AccountKeeper,
```

Comment:
Can you remove the deref in the `app.LockupKeeper` init right above this?

Reply:
Fixed!